indexer/run.sh

```diff
@@ -98,26 +80,26 @@ index() {
 sed -i '/if package_exists snapcraft/,/fi/d' ./build/install-build-deps.sh
 ./build/install-build-deps.sh
 
-index linux 'target_os="linux"'
+index linux 'target_os="linux"' | true
```

Comment:
why not just `set +e` at the top of index and `set -e` at its exit?

Reply:
Why? I think we want `set -e` inside the index, don't we (hence, the opposite)?

So, the problem we're solving here:

1. commands in the `index()` are sequential and each steps requires the previous one to succeed (otherwise we'd have 300 byte index artifacts that are invalid). later commands shouldn't be executed if any step inside the `index()` fails
2. the `index()` overall can fail and it's still not a complete failure since we'd want to index the other configs

We already have `set -e` on top, the only problem is that we want 2. to be solved. This is exactly the workaround for this.